src/components/left-pane/RelatedProperties.js

```diff
@@ -21,39 +21,45 @@ const RelatedProperties = ({ property }) => {
 
   const gotoProperty = () => {
     dispatch(setLngLat(lng, lat));
+    handlePropertyClick();
```

Comment:
`handlePropertyClick()` toggles the highlighting of a property. I think in this scenario, we want to always highlight it.
So I think it makes sense for this line to be `dispatch(selectRelatedProperties({ [property.poly_id]: property }));` instead

Reply:
Thanks for the knowledge, man. All sorted.